Documentation/gettingstarted/clustermesh/aks-clustermesh-prep.rst

```diff
@@ -0,0 +1,235 @@
+.. _gs_clustermesh_aks_prep:
+
+**********************************
+AKS-to-AKS clustermesh preparation
+**********************************
+
+This is a step-by-step guide on how to install and prepare 
+AKS(Azure Kubernetes Service) clusters in BYOCNI mode to meet the requirements 
+for the clustermesh feature.
+
+In this guide we will install two AKS clusters in BYOCNI(Bring Your Own CNI) 
+mode and connect them together via clustermesh. This guide is not 
+applicable for cross-cloud clustermesh since 
+
+.. note::
+
+        BYOCNI requires the ``aks-preview`` CLI extension with version >=
+        0.5.55, which itself requires an ``az`` CLI version >= 2.32.0 .
+
+Install cluster one
+###################
+
+1. create a resource group for the cluster(or set the environment variables
+to an existing resource group).
```

Comment:
Pretty sure this should be indented (aligned on “Create”) for the numbered list to be rendered properly. Have you tried building the doc locally? Same comment applies to your other list items throughout the document.

Reply:
You are right. It did work, but it triggered a warning in the linter, for which I added an exception because it wasn't clear this was the issue. Fixed it, and not able to remove the linter exception. Thx!